### ISSUE TITLE
comparing KSM motif scan scores with PWM and TFFM scores in a ROCs

### ISSUE THREAD

**Issue by RomainBlancMathieu** [yguoma/GEM3#22]
Hi, I am comparing KSM scores with scores of a single PWM matrix and a single TFFM matrix in a ROC for a given plant TF.
I am not sure how to deal with the scores output in the KSM motif scan output. I did as follow:
For each searched sequences (for a positive and for a negative test set) I collect scores for all kmer in all motifs (KMAC detected 13 motif in the training set, n=800, DAPseq peaks).
then I take either the sum of all scores for a given searched sequence, or the mean or just keep the best.
Then I compare the curve and AUC for the KSM, PWM and TFFM (see plots below).
My first question is: is it correct to compute the sum, mean or best as I do?
Second question: is it correct to then compare these sum, mean or max KSM score against the PWM or TFFM scores obtained with a single matrix?
![comparison_kmerScoresMeth](https://user-images.githubusercontent.com/58161730/110443059-d1153600-80bb-11eb-93e4-8ad18ed4725c.png)
